openghg/standardise/meta/_metadata.py

```diff
@@ -33,6 +33,22 @@ def metadata_default_keys() -> List:
     return default_keys
 
 
+def metadata_keys_as_floats() -> List:
+    """
```

Comment:
```
   """Define which keys should be consistently stored as numbers in the metadata
    (even if they are not numbers within the attributes).

    Returns:
        list: List of keys to convert to floats

    """
```

Reply:
Hmm, not sure how this commit change suggestion thing works.

Reply:
I believe the change suggestion only allows one line to be changed.

I'm a little confused about the suggested change here? This should have passed all the linting already?